### PR TITLE
feat(widgets): add support for color picker in actions

### PR DESCRIPTION
**Pull request by ghusse**
Linked to CU-85yxm8nr1

### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

File: packages/agent/test/utils/forest-schema/generator-action-field-widget.test.ts [hunk 1]
```diff
@@ -512,16 +512,45 @@ describe('GeneratorActionFieldWidget', () => {
...
-      expect(() => {
-        GeneratorActionFieldWidget.buildWidgetOptions({
+    describe('ColorPicker', () => {
+      it('should return a valid widget edit with default values', () => {
+        const result = GeneratorActionFieldWidget.buildWidgetOptions({
           type: 'String',
           label: 'Label',
           watchChanges: false,
-          // @ts-expect-error Unsupported widget
-          widget: 'UnsupportedWidget',
```
Comment (Thenkei) [inline packages/agent/test/utils/forest-schema/generator-action-field-widget.test.ts:522]: Why did you remove this test ?

Comment (ghusse) [reply packages/agent/test/utils/forest-schema/generator-action-field-widget.test.ts:522]: it's an error after a merge I suppose. I'll put it back.

**Review by Thenkei — APPROVED** [review]
